scripts/scil_visualize_bingham_fit.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Visualize 2-dimensional Bingham volume slice loaded from disk.
```

Comment:
This volume is assumed to be saved from scil_compute_bundle_specific_fodf_metrics.py

Reply:
Done in d989322 :+1: